tests/cypress/e2e/menuActions/delete.cy.ts

```diff
@@ -32,7 +32,7 @@ describe('delete tests', () => {
         jcontent.getAccordionItem('pages').getTreeItem('test-pageDelete1').contextMenu().select('Delete');
 
         cy.log('Verify dialog opens and can be mark for deletion');
-        getComponent(DeleteDialog).markForDeletion('You are about to delete 5 items, including 3 page(s)');
+        getComponent(DeleteDialog).toggleRowExpanded().markForDeletion('You are about to delete 5 items, including 3 page(s)');
```

Comment:
I added toggleRow for debugging as previous runs was showing 6 items instead of 5. Not able to reproduce anymore, but keeping for next time. 

Earliest instance of failure I see so far was from this run https://github.com/Jahia/jcontent/actions/runs/15856427628/job/44703255994